ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot

```diff
@@ -24,34 +28,32 @@ Uninstall RHODS In OSD
   Log To Console   ${output}
 
 Uninstall RHODS In Self Managed Cluster
-      [Documentation]  Uninstall rhods from self-managed cluster
-      IF  "${INSTALL_TYPE}" == "CLi"
-          Uninstall RHODS In Self Managed Cluster Using CLI
-      ELSE IF  "${INSTALL_TYPE}" == "OperatorHub"
-          Uninstall RHODS In Self Managed Cluster For Operatorhub
-      ELSE
-           FAIL    Provided install type is not supported
-      END
-
+  [Documentation]  Uninstall rhods from self-managed cluster
+  IF  "${INSTALL_TYPE}" == "CLi"
+      Uninstall RHODS In Self Managed Cluster Using CLI
+  ELSE IF  "${INSTALL_TYPE}" == "OperatorHub"
+      Uninstall RHODS In Self Managed Cluster For Operatorhub
+  ELSE
+        FAIL    Provided install type is not supported
+  END
 
 RHODS Operator Should Be Uninstalled
   Verify RHODS Uninstallation
   Log  RHODS has been uninstalled  console=yes
 
 Uninstall RHODS In Self Managed Cluster Using CLI
-   [Documentation]   UnInstall rhods on self-managedcluster using cli
-   Clone OLM Install Repo
-   ${return_code}    ${output}    Run And Return Rc And Output   cd ${EXECDIR}/${OLM_DIR} && ./cleanup.sh -t operator   #robocop:disable
-   Should Be Equal As Integers	${return_code}	 0   msg=Error detected while un-installing RHODS
-   Log To Console   ${output}
+  [Documentation]   UnInstall rhods on self-managedcluster using cli
+  Clone OLM Install Repo
+  ${return_code}    Run and Watch Command    cd ${EXECDIR}/${OLM_DIR} && ./cleanup.sh -t operator    timeout_min=20
+  Should Be Equal As Integers	${return_code}	 0   msg=Error detected while un-installing RHODS
 
 Uninstall RHODS In Self Managed Cluster For Operatorhub
-   [Documentation]   Uninstall rhods on self-managed cluster for operatorhub installtion
-   ${return_code}    ${output}    Run And Return Rc And Output   oc create configmap delete-self-managed-odh -n redhat-ods-operator
-   Should Be Equal As Integers	${return_code}	 0   msg=Error creation deletion configmap
-   ${return_code}    ${output}    Run And Return Rc And Output   oc label configmap/delete-self-managed-odh api.openshift.com/addon-managed-odh-delete=true -n redhat-ods-operator
-   Should Be Equal As Integers	${return_code}	 0   msg=Error observed while adding label to configmap
-   Verify Project Does Not Exists  redhat-ods-applications
-   Verify Project Does Not Exists  redhat-ods-monitoring
-   Verify Project Does Not Exists  rhods-notebooks
-   ${return_code}    ${output}    Run And Return Rc And Output   oc delete namespace redhat-ods-operator
+  [Documentation]   Uninstall rhods on self-managed cluster for operatorhub installtion
+  ${return_code}    ${output}    Run And Return Rc And Output   oc create configmap delete-self-managed-odh -n redhat-ods-operator
```

Comment:
wouldn't we need to put `Run and Watch Command` here too?

Reply:
We should defently start to use "Run and Watch Command" everywhere in RF if a command takes more than few seconds. 
Is that the case here ?

Reply:
I've also added it to the RHODS CLI install test, here's an an output example:
http://10.0.146.152:8080/job/rhods-ci-pr-test/12/console

Reply:
> Is that the case here ?

hm probably not